style.css

```diff
@@ -0,0 +1,245 @@
+*{
+	box-sizing: border-box; 
+}
+html{
+	font-size: 20px; 
+}
+body {
+	background-color:#771668;
+	width: 100%;
+	margin-top:80px;
+	margin-left:0px;
+	margin-right:0px;
+	margin-bottom:0px;
+}
+
+header{
+	position: fixed;
+	top:0px;
+	left:0px;
+	background-color: #2d094f;
+	width:100%;
+	padding: 10px;
+	border:3px solid black;
+	z-index:2;
```

Comment:
for z-index, it is advised to increment it by e.g. 10 at a time, so that in future if you wanted to slot a layer in between you wouldn't have to change all the existing z indexes